cmake/macros/SundialsAddLibrary.cmake

```diff
@@ -325,10 +325,17 @@ macro(sundials_add_library target)
 
       # set the correct output name
       if(sundials_add_library_OUTPUT_NAME)
-        set_target_properties(${_actual_target_name} PROPERTIES
-          OUTPUT_NAME ${sundials_add_library_OUTPUT_NAME}
-          CLEAN_DIRECT_OUTPUT 1
-        )
+        if(WIN32 AND ${_libtype} MATCHES "STATIC")
```

Comment:
Consider changing this to the following:
``` suggestion
        if((MSVC OR ("${CMAKE_C_SIMULATE_ID}" STREQUAL "MSVC")) AND ${_libtype} MATCHES "STATIC")
```
For MinGW, `WIN32` is true. But it doesn't have the naming conflict that compilers that target MSVC have.

Reply:
Thanks for your help! I understood from @balos1 that this PR should be closed in favor to a more generic/customizable approach.
Let me know!